examples/shallow_water/williamson2.py

```diff
@@ -84,7 +83,7 @@
 
 # nonlinear swe forms
 
-def form_function(u, h, v, q):
+def form_function(u, h, v, q, t):
```

Comment:
Thanks for updating this script. Please could you update the other examples too?